src/components/connection_handler/test/connection_handler_impl_test.cc

```diff
@@ -407,7 +407,7 @@ TEST_F(ConnectionHandlerTest, IsHeartBeatSupported) {
 TEST_F(ConnectionHandlerTest,SendEndServiceWithoutSetProtocolHandler) {
   AddTestDeviceConnection();
   AddTestSession();
-  protocol_handler_test::ProtocolHandlerMock mock_protocol_handler;
+  protocol_handler_test::MockProtocolHandler mock_protocol_handler;
```

Comment:
@Kozoriz May be move this line to SetUp() ? You are using the same code in lots of test cases.